bento-sample-app/src/main/res/layout/activity_tab_viewpager.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is the same layout as `bento_component_tab_view_pager.xml`. We should just use that instead of duplicating it.